packages/theme/pages/Checkout/Shipping.vue

```diff
@@ -18,81 +18,53 @@
     />
     <form @submit.prevent="handleSubmit(handleFormSubmit)">
       <div v-if="!selectedSavedAddressId" class="form">
-        <ValidationProvider
-          v-if="!isAuthenticated"
-          name="email"
-          rules="required|email"
-          v-slot="{ errors }"
-          slim
-        >
-          <SfInput
-            v-on:click="getBackToShippingDetails()"
-            :class="{'disable-input': isFormSubmitted}"
+        <EmailValidationProvider v-slot="{ errors }" v-if="!isAuthenticated" slim append-rules="required|trim">
```

Comment:
Make sure that `trim` and `slim` isn't something we'd like to have by default in the `EmailValidationProvider`. It's alright if not, however, customization of these reusable components should be as minimal as possible, so keep that in mind.

Reply:
The `slim` prop is being used in all the places where `ValidationProvider` is used so I think we can go ahead and make it a default rule.  It's not the same with `trim` but I think it's useful for all the use cases to have it applied.